### PR TITLE
Show hide menu

### PR DESCRIPTION
**Pull request by hrodberaht**

### CODE REVIEW

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import React, { Component } from 'react';
-import SideBar from './SideBar';
+import ConnectedSideBar from './SideBar';
```
Comment (pokiujf) [inline src/components/Header/Header.js]: Naming connected components differently is not a common practice, and we do not get a lot of value from it.

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
 export default class Header extends Component {
   render() {
-    return <SideBar />;
+    return <ConnectedSideBar />;
```
Comment (pokiujf) [inline src/components/Header/Header.js]: If there are no plans to have more markup in header, than this component is obsolete and we can use Sidebar directly

File: src/store/navigation/reducer.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import * as types from './types';
+
+const initialState = {
+  toggleSidebar: true,
+};
+
+const navigation = (state = initialState, action) => {
+  switch (action.type) {
+    case types.TOGGLE_SIDEBAR:
+      return { ...state, toggleSidebar: !state.toggleSidebar };
```
Comment (pokiujf) [inline src/store/navigation/reducer.js]: `toggleSidebar` is name of the action, not the state the sidebar is in.
what about
```
case types.TOGGLE_SIDEBAR:
      return { ...state, isSidebarVisible: !state.isSidebarVisible };
```

File: package.json [hunk 4]
```diff
@@ -51,7 +53,9 @@
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.2",
     "eslint-plugin-react": "^7.11.1",
-    "nodemon": "^1.18.5"
+    "nodemon": "^1.18.5",
```
[the remark this answers is not in the document]
Comment (hrodberaht) [reply package.json]: I use nodemon for server.js.

**Pull request merged**